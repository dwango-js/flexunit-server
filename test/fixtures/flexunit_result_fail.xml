<testcase classname='jp.nicovideo::Test1'
          name='test_fail'
          time='0.000'
          status='failure'>
  <failure message='expected:A but was:B'
           type='jp.nicovideo::Test1.test_fail' >
    <![CDATA[]]>
  </failure>
</testcase>
