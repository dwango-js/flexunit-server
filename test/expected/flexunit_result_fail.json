{
  testcase: [
    {'$': {
      classname: "jp.nicovideo::Test1",
      name: "test_fail",
      status: "failure",
      time: "0.000"
    },
     failure: [
       {'$': {message: "expected:A but was:B",
              type: "jp.nicovideo::Test1.test_fail"
             }}
     ]
    }
  ]
}
